### DISCUSSION TITLE
Bed tramming menu - center option missing

### DISCUSSION THREAD

**Discussion by JALTAV** [mriscoc/Ender3V2S1#1101] [General]
Thanks for the excellent work.

after installing the latest firmware (20230904), the center option is misssing.

**Comment by mriscoc** [comment]
Hi, for bed tramming in Ender printers there is no need to test in the center, but that location can be enabled for manual bed tramming by enabling BED_TRAMMING_INCLUDE_CENTER and building a custom version.

**Reply by JALTAV** [reply]
Thank you Miguel for the tip.
